source/containers/Form/hooks/formReducer.js

```diff
@@ -1,16 +1,42 @@
-const actions = {
-  increaseCounter: 'INCREASE_COUNTER',
-  decreaseCounter: 'DECREASE_COUNTER',
+import { increaseCount, decreaseCount } from '../../../helpers/Counter';
+
+const INCREASE_COUNTER = 'INCREASE_COUNTER';
+const DECREASE_COUNTER = 'DECREASE_COUNTER';
+
+export const actions = {
```

Comment:
All actions can be in a separate file. Then formReducer would look more clean with only managing states.